test/syscalls/linux/tuntap.cc

```diff
@@ -490,5 +491,25 @@ TEST_F(TuntapTest, WriteHangBug155928773) {
   write(sock, "hello", 5);
 }
 
+// EmptyInterfaceName tests empty interface name at creation of tun/tap.
+TEST_F(TuntapTest, EmptyInterfaceName) {
+  SKIP_IF(!ASSERT_NO_ERRNO_AND_VALUE(HaveCapability(CAP_NET_ADMIN)));
+
+  // Interface creation.
+  ASSIGN_OR_RETURN_ERRNO(FileDescriptor fd, Open(kDevNetTun, O_RDWR));
+
+  struct ifreq ifr_set = { 0 };
```

Comment:
is `{ 0 }` necessary? `{}` should be ok

Reply:
Both are equivalent semantically.
I personally like the `{ 0 }`, more explicit convention, better.
It is more pronounce in line `508` (struct assignment).

Reply:
Do you agree❓

Reply:
There's a lot of `= {};` in the codebase, I think it's best to stay consistent.

Reply:
Fixed.